src/main/java/frc/robot/FinalContainer.java

```diff
@@ -73,14 +80,22 @@ private void configureButtonBindings() {
 
     // JoystickButton button = new JoystickButton(m_controller, XboxController.k.value);
     buttonBack.whenPressed(m_drivetrainSubsystem::zeroGyroscope);
-    buttonX.whenPressed(new CatapultDouble(m_CatapultLeftSubsystem, m_CatapultRightSubsystem, 2));
+    // buttonX.whenPressed(new CatapultDouble(m_CatapultLeftSubsystem, m_CatapultRightSubsystem, 2));
     //buttonB.whenPressed();
-    buttonA.whenPressed(new CatapultDouble(m_CatapultLeftSubsystem, m_CatapultRightSubsystem,1 ));
-    buttonY.whenPressed(new CatapultDouble(m_CatapultLeftSubsystem, m_CatapultRightSubsystem, 2));
+    // buttonA.whenPressed(new CatapultDouble(m_CatapultLeftSubsystem, m_CatapultRightSubsystem,1 ));
+    // buttonY.whenPressed(new CatapultDouble(m_CatapultLeftSubsystem, m_CatapultRightSubsystem, 2));
     buttonLBumper.whenPressed(new ToggleIntake(m_IntakeMotors));
     buttonRBumper.whenPressed(new Spit(m_IntakeMotors));
     buttonLeftJoystickButton.whenPressed(new AutoMeter(m_drivetrainSubsystem, m_CatapultLeftSubsystem, m_CatapultLeftSubsystem, m_IntakeMotors));
     buttonRightJoystickButton.whenPressed(new AutoCommand(m_drivetrainSubsystem));
+
+
+    // limelight commands
+    buttonB.whenPressed(new GoToDistanceLimelight(70, m_drivetrainSubsystem, ll));
+    buttonA.whenPressed(new SwitchLimelightStream());
+    buttonX.whenPressed(new TurnToZeroLimelight(0, m_drivetrainSubsystem, ll));
+    buttonY.whenPressed(new AlignAndMoveToLimelight(60, m_drivetrainSubsystem, ll));
+
```

Comment:
worth going through and setting which buttons do which, imported the necessary commands for limelight and stole some buttons from other commands in order to test limelight